LandSandBoat/server
# Imp Skills

**_I affirm:_**

## What does this pull request do?

This implements the missing Imp skills: Stifling Tantara and Grating Tantara. Along side it there are some small adjustments such as reducing the damage mod for Frenetic Rip according to the Monster Stats spreadsheet by Jimmayus. https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395

## Steps to test these changes

1. Go fight Zikko `!gotoid 17101144`
2. If it is not up then `!spawnmob 17101144`
3. Once fighting it use `!tp 3000` to force it to use TP moves and see that it correctly does these new TP moves instead of the single target ones.
